comp/core/agenttelemetry/impl/agenttelemetry_test.go

```diff
@@ -476,7 +477,8 @@ func TestRun(t *testing.T) {
 	// The order is not deterministic
 	profile0Len := len(r.(*runnerMock).jobs[0].profiles)
 	profile1Len := len(r.(*runnerMock).jobs[1].profiles)
-	assert.True(t, (profile0Len == 1 && profile1Len == 3) || (profile0Len == 3 && profile1Len == 1))
+	t.Logf("%+v", r.(*runnerMock).jobs)
```

Comment:
is this a left over debug statement or intentional?

Reply:
Kind of both ?
I had a hard time debugging this test (it's not clear which entries of the config end up in which "profile"), the log helps a lot 😄
So I think it's worth keeping